### DISCUSSION TITLE
what color are you Wooshing with?

### DISCUSSION THREAD

**Discussion by godbout** [godbout/Wooshy.docs#65] [General]
i started with the `yellow` and i liked it a lot. but sometimes it's hard to see. like on white backgrounds.
then i've switched to `sharp green`. looked nice for a while, but i don't know, also some contrast issues i think.
then.. .i tried... `the pink`. and i can't go back. i've tried. i've tried yellow again, then green, then even some `blue`...

but no. `the pink` wins. always and forever.

![Screenshot 2022-08-26 at 01 25 38](https://user-images.githubusercontent.com/121373/186730425-81138790-e3ae-4e4e-9eee-0c872a62b987.png)

so. what's your color?

**Comment by dudunato** [comment]
Ok, now I'm belong to the pink team 💕

**Reply by godbout** [reply]
yes you are belong to us.

**Comment by aaronrolls** [comment]
I can’t say I’m on team pink. I prefer whatever colour I have set in MacOS at the time.

But for a default colour, have you tried baby blue?

**Comment by godbout** [comment]
@aaronrolls yes. @dudunato was blue and he tried to change me and we wrestled a bit and now we're both pink.

if i were you i would still try a bit of pink. for one or two or some hundred days. just to really, really confirm it's not your thing.

**Comment by aaronrolls** [comment]
Maybe if I could get pastel pink I could get on board.
![pastel-pink-color-solid-background-1920x1080](https://user-images.githubusercontent.com/47949221/186999640-21071539-441a-4a5d-94bc-78c83f9ea3ae.png)

**Comment by godbout** [comment]
@aaronrolls you got it.
also, you probably don't need to know that, but every time i read your name i think of the [Baltimore Accent](https://www.youtube.com/shorts/lfIWX5vGTEk).

**Reply by aaronrolls** [reply]
Uncanny. That is exactly what I sound like.

**Comment by godbout** [comment]
`Aaron Pastel Pink` color added for the Targets in a20: https://github.com/godbout/Wooshy.docs/releases/tag/1.a20

**Reply by aaronrolls** [reply]
Pastel pink is better then normal pink for sure. Still not convinced it is better than baby blue however. But it is pretty close.
<img width="912" alt="CleanShot2022-08-28at13 27 12@2x" src="https://user-images.githubusercontent.com/47949221/187053699-89921e98-3ba2-4a65-be53-6ca5512221b8.png">

**Reply by godbout** [reply]
yeah tbh i wasn't very impressed by the Pastel Pink. baby blue is already available through the hue slider?
